### DISCUSSION TITLE
Classic Combat

### DISCUSSION THREAD

**Discussion by xJon** [ACGaming/UniversalTweaks#171] [Tweak Suggestions]
https://legacy.curseforge.com/minecraft/mc-mods/classic-combat

Would be appreciated to also remove the attack speed from weapon tooltips which are confusing:
![image](https://user-images.githubusercontent.com/8945347/235714682-c1ff2da3-9d16-46c5-99f3-715d4ff34be8.png)
and perhaps also the weapon cooldown animations?

**Comment by Ski-z** [comment]
In the meantime, you can use CraftTweaker for this.

`<minecraft:stone_sword:*>.removeTooltip("Attack Speed");` will work.

**Reply by xJon** [reply]
That's true! Although there are many weapons in a modpack, and they can change from update to update, so it doesn't feel like an ideal solution.

**Comment by ACGaming** [comment]
Implemented in 96c7650e25134e174c314d4e1a6ef09edd6744aa.
